app/templates/_has_wants_selector.html

```diff
@@ -0,0 +1,182 @@
+            PPE Types You {{ title }}
+            <button data-target="#{{ name }}" class="btn btn-primary no-edit start-edit"><span
+                    class="glyphicon glyphicon-pencil" aria-hidden="true"></span> Edit</button>
+        </h2>
+        <table class="table table-striped">
+            <thead>
+                <tr>
+                    <th>Image</th>
+                    <th>SKU</th>
+                    <th>Description</th>
+                    <th>Quantity Wanted</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr class="only-edit" id="add-ppe-row-{{ name }}">
+                    <td colspan="4">
+                        <div
+                            style="display: flex; width: 100%; align-items: center; justify-content: space-between; flex-direction: row;">
+                            <button class="btn btn-default"
+                                onclick="window.document.location.reload(); return false">Cancel</button>
+                            <button class="btn btn-success start-add-ppe" data-target="{{ name }}">
+                                <i class="glyphicon glyphicon-plus" aria-hidden="true"></i> Add PPE Type
+                            </button>
+                            <button class="btn btn-primary">Save</button>
+                        </div>
+                    </td>
+                </tr>
+                {% for ppe in ppe_types %}
+                {% if ppe[name]|length > 0 %}
+                <tr>
+                    {{ active_ppe_row(ppe) }}
+                </tr>
+                {% endif %}
+                {% endfor %}
+            </tbody>
+        </table>
+    </form>
+</div>
+{% endmacro %}
+
+{% block has_wants_selector %}
+<div class="modal fade" tabindex="-1" role="dialog" id="add-ppe-modal">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span
+                        aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title">Add a PPE Type</h4>
+            </div>
+            <div class="modal-body">
+                <input type="search" style="width: 100%" class="form-control" placeholder="Search for a PPE Type" id="add-ppe-type-search"/>
+                <br/>
+                <table class="table table-striped" id="add-ppe-types">
+                    <thead>
+                        <tr>
+                            <th>Image</th>
+                            <th>SKU</th>
+                            <th>Description</th>
+                            <th>Actions</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for ppe in ppe_types %}
+                        {% if ppe.wants|length > 0 or ppe.has|length > 0 %}
+                            {% set hidden = "hidden" %}
```

Comment:
When the user opens the new PPE type popup, we don't want them to initially see a bunch of types they already have/want. However, if they search for a type they already have/want, we want it to show up. The easy solution to this is to simply have all the PPE types the user already has/wants. The search script doesn't care and will un-hide them as soon as the user starts typing.

The only downside of this is that if the user searches for something then deletes their query, all the types they already have/want will still be there. That feels like a fine tradeoff to me.